generators/entity/templates/src/main/webapp/app/entities/_entity-management.js

```diff
@@ -16,7 +16,31 @@ angular.module('<%=angularAppName%>')
                         controller: '<%= entityClass %>ManagementController'
                     }
                 },
-                resolve: {<% if (enableTranslation){ %>
+                <%_ if (pagination == 'pagination' || pagination == 'pager'){ _%>
+                params: {
```

Comment:
These are used to set the defaults for each param and guarantee they aren't undefined. For squash, it means that the param won't show in the url if it is set to the default - so if page is 1 you won't see `page=1`.